package.json

```diff
@@ -57,5 +67,17 @@
   },
   "engines": {
     "node": ">=6"
+  },
+  "dependencies": {
+    "vue-demi": "^0.13.2"
+  },
+  "peerDependencies": {
+    "@vue/composition-api": "^1.0.0",
+    "vue": "^2.6.0 || >=3.2.0"
```

Comment:
In theory yes, but `vue-demi` only supports `^2.6.0` and since we depend on it, I opted per matching that limit. 
On the `3.2.0` I don't actually remember why I did it 🤔 , maybe I had some errors while running the tests on prior v3 version, but will re-investigate.